app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/CellPlotDaoIT.java

```diff
@@ -168,6 +170,21 @@ void fetchExpressionPlotWithInvalidGeneIdHasNoExpression(String experimentAccess
                 .allSatisfy(dto -> assertThat(dto).hasFieldOrPropertyWithValue("expressionLevel", 0.0));
     }
 
+    @Test
+    void canFetchDefaultPlotTypeAndOption(){
```

Comment:
A suggestion name for the test `fetchDefaultPlotMethodAndOptionWithExistingExperimentAccession`.